client/src/components/Connections/scenes/Results/components/InfoLoader/InfoLoader.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import React from 'react'
+import {compose} from 'redux'
+import {connect} from 'react-redux'
+import {withDataProviders} from 'data-provider'
+
+import {connectionEntityDetailProvider} from '../../../../../../dataProviders/connectionsDataProviders'
+// import Info from '../info/Info'
+import './InfoLoader.css'
+
+const InfoLoader = (props) => (
+  <div
+    className={
+      props.recursive
+        ? 'infoWrapper'
+        : 'infoWrapper col-md-offset-2 col-md-8 col-lg-offset-3 col-lg-6'
+    }
+  >
+    {<div>{props.data.entities[0].entity_name}</div>}
+    {/* <Info data={props.data} eid={props.eid} /> */}
```

Comment:
I assume this is the componetn overlap with the part @MarekCacko was doing (just checking, in that case it's fine)

Reply:
Yup, I hope so :slightly_smiling_face: @MarekCacko can you please confirm?

Reply:
Yes, everything what you had in Info directory is copy of components from my code. Even this InfoLoader.js was in my code (now it has a new name)